package.json

```diff
@@ -13,6 +13,7 @@
     "env-cmd": "^10.1.0",
     "file-saver": "^2.0.5",
     "html2canvas": "^1.0.0-rc.7",
+    "http-status-codes": "^2.1.4",
     "i18next": "^19.8.7",
```

Comment:
not required, plz remove

Reply:
copy this instead
https://github.com/prettymuchbryce/http-status-codes/blob/master/src/status-codes.ts

Reply:
done